.circleci/config.yml

```diff
@@ -16,10 +20,10 @@ references:
 
 # Container for all the builds
 executors:
-  commit-executor:
+  build-executor:
     docker:
-      - image: broadinstitute/study-website-build:04-09-2020A
-    working_directory: ~/repo/ddp-workspace
+      - image: broadinstitute/study-website-build:05-06-2020A
+    working_directory: *ng_workspace_path
     resource_class: medium+
```

Comment:
Using the new "constant" workspace path location. ddp-workspace is where most of the action occurs, so might as well make it the default.